python/looker/rtl/serialize.py

```diff
@@ -7,7 +7,6 @@
 from typing import Callable, Dict, List, Type, Union
 
 import cattr
-
```

Comment:
huh, yapf really shouldn't kill this line. there should be a single blank line between groupings of imports (stdlib, 3rd-party, own-lib/app)